failover-handler.py

```diff
@@ -233,6 +253,12 @@ def main():
             log('Detach interface done')
         elif event in ['member-leave', 'member-failed']:
             for member in members:
+                log("Member reported left/failed, ip=%s, az=%s" % (member.ip, member.az))
+                if not is_member_down(member.ip):
```

Comment:
If I understand correctly, here we would be waiting 3*5 seconds before we attach the interface in case the machine is actually down.